fbw-a32nx/src/systems/instruments/src/ND/style.scss

```diff
@@ -236,3 +236,11 @@ path.rounded {
   stroke-linecap: round;
   stroke-linejoin: round;
 }
+
+.hidden {
+  display: none;
+}
+
+.a380xInvisible {
+  display: block;
```

Comment:
I don't think this should force block. Do we even need any style for it?

Reply:
empty rulesets are not allowed, or do you mean just not defining it in a32nx's style.scss? Don't know how that behaves. I would do `display: inherit` now

Reply:
That would also modify the behaviour. Why do we need a style in the A32NX CSS for this at all?

Reply:
Hmm yeah you're right. There's no a32nx/a380x prop passed down or defined anywhere, so I found this the best way to have an aircraft-specific style/visibility, but I'm open to other ideas. 
I could leave the style empty, although then lint complains

Reply:
You can just not have it in the a32nx stylesheet at all.

Reply:
Fixed that now